src/framework/theme/components/tree-grid/data-source/tree-grid.model.ts

```diff
@@ -4,52 +4,33 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-/**
- * Table's data interface
- */
-export interface NbTreeGridNode<T> {
-  /**
-   * Data object which will be available as a context of rows and cell templates
-   * @type T
-   */
-  data: T,
-  /**
-   * Child rows
-   */
-  children?: NbTreeGridNode<T>[];
-  /**
-   * Row expand state
-   */
-  expanded?: boolean;
-}
-
 export const DEFAULT_ROW_LEVEL: number = 0;
```

Comment:
The last thing. Let's add *Nb* prefix for *DEFAULT_ROW_LEVEL*, *DataGetter*, *ChildrenGetter*, *ExpandedGetter* and *Getters*.

Reply:
Done in https://github.com/akveo/nebular/pull/1254/commits/e51c46afd5d28a81d8e9505021084bd5780ec004. Thanks!